examples/hello-electron/main.ts

```diff
@@ -0,0 +1,38 @@
+import { app, BrowserWindow, ipcMain } from 'electron';
+import { init, debug, generateDeviceCode } from '@bitdrift/electron';
+import { config as configDotenv } from 'dotenv';
+
+configDotenv();
+
+// Initialize bitdrift
+const logger = init(process.env['BITDRIFT_KEY'] ?? 'invalid key', {
+  url: process.env['BITDRIFT_URL'],
+  appVersion: '1.0.0',
+});
```

Comment:
Looks like this is missing the session strategy, probably a case of stale dist folder.